clusterloader2/api/types.go

```diff
@@ -0,0 +1,83 @@
+package api
+
+import (
+	"time"
+)
+
+type Config struct {
+	// Number of namespace in the test.
+	Namespaces int32 `json: Namespaces`
```

Comment:
the convention is to start json tags with lower letters.

Reply:
done.